MusicStore.Web/Controllers/HomeController.cs

```diff
@@ -1,11 +1,20 @@
-﻿using MusicStore.Business.Interfaces;
+﻿using System.Linq;
+using System.Security.Claims;
 using System.Web.Mvc;
 namespace MusicStore.Web.Controllers
 {
     public class HomeController : Controller
     {
         public ActionResult Index()
         {
+            var userIdentity = (ClaimsIdentity)User.Identity;
+            var claims = userIdentity.Claims;
+            var roleClaimType = userIdentity.RoleClaimType;
+            var roles = claims.Where(c => c.Type == roleClaimType).ToList();
+            if (roles.Count == 1)
```

Comment:
Change to be Any()

Reply:
I also need to be sure that user has only 1 role